src/utils/Validator.ts

```diff
@@ -42,10 +41,10 @@ const validateDimensionSet = (dimensionSet: Record<string, string>): void => {
   Object.entries(dimensionSet).forEach(([key, value]) => {
     dimensionSet[key] = value = String(value);
 
-    if (!validator.isAscii(key)) {
+    if (key.match(Constants.VALID_DIMENSION_REGEX) == null) {
```

Comment:
I would use `test` instead of `match` here

```
const paragraph = 'ab[cd';
const regex = /[a-z]/g;
const found = paragraph.match(regex);

console.log(found); // won't be null even though paragraph has an invalid string
```

Reply:
Done, thanks for the suggestion!